### ISSUE TITLE
In the gnome-shell calendar, days of a different month with events are colored differently from those without

### ISSUE THREAD

**Issue by Raytar** [ubuntu/yaru#2164]
![image](https://user-images.githubusercontent.com/10762082/80280572-24310500-8705-11ea-9627-3eaee8fd0383.png)

##  Expected Behavior
Any day of a different month should be colored differently from the days of the current month. 

## Actual Behavior
Days of a different month that have events are colored the same as days of the current (selected) month (January 1. and 8. in the image above have the wrong color).

## Steps to Reproduce the Problem

  1. Add events to the calendar in the beginning or end of a month
  2. Open the calendar and look at the colors of the dates

## Software that presents the issue

- Name: gnome-shell
- Version: 3.36.1-5ubuntu1

## Yaru version

```
$ apt show yaru-theme-gnome-shell 
Package: yaru-theme-gnome-shell
Version: 20.04.6
```

**Comment by clobrano** [comment]
Hey @Raytar, thanks for taking time to contribute, and good catch

**Comment by clobrano** [comment]
This might be an upstream issue, but I have to wait until my new modem arrives to have connection enough to download an up to date fedora for testing :)

**Comment by clobrano** [comment]
Alright, no Fedora32 doesn't have this specific problem, but they have what we have fixed already :D, so, let's try to fix it locally, then in a next upstream sync will see what to do

**Comment by clobrano** [comment]
Analyzing this issue I found two problems:

1. non-workdays with appointments have a wrong color (the same as the workdays, instead than paler).
2. next-month-days with appointments have a wrong color (the same as the workdays, instead than paler): this is the bug this ticket was opened for.

Focal 20.04 only has (2), while groovy 20.10 only has (1), which we had fixed in 20.04. So the plan is

- to look for the fix in Focal and check why isn't working in groovy
- check the fix in Groovy and backport it in Focal
